### DISCUSSION TITLE
Hide the appender at the root level in the Site Editor

### DISCUSSION THREAD

**Discussion by jameskoster** [WordPress/gutenberg#36453] [Ideas]
There are a few issues and PRs detailing issues and exploring potential solutions surrounding the appender inserter.

* https://github.com/WordPress/gutenberg/issues/26404
* https://github.com/WordPress/gutenberg/pull/36037
* https://github.com/WordPress/gutenberg/pull/31886

It seems the dragons become more numerous when handling containers and nested containers. 

I'm wondering if it might be better to break things down and address them one at a time. For example, the inserter visible here when editing a header in template part focus mode is especially awkward:

<img width="1646" alt="Screenshot 2021-11-12 at 16 12 53" src="https://user-images.githubusercontent.com/846565/141504304-0682ef38-87ff-49b4-90bb-892af63c07d5.png">

But also relatively trivial to address (famous last words). We could try simply hiding the appender at the root level in the site editor whenever one or more blocks are present:

<img width="1169" alt="Screenshot 2021-11-12 at 16 54 25" src="https://user-images.githubusercontent.com/846565/141504684-67d66d85-5bd5-4bbd-8cea-0293e451f2b8.png">

Worth a try?

**Comment by jasmussen** [comment]
Worth a try, yes.

**Comment by shaunandrews** [comment]
There was some discussion in #36827 — the general consensus was that we don't need the appender when editing a template part.
